modules/distribution/src/main/scripts/local-fluo

```diff
@@ -66,7 +68,7 @@ function check_dirs {
 }
 
 function kill_if_running {
-  validate_app
+  validate_app_full
```

Comment:
@mikewalch  while testing this I ran into an issue with `validate_app` not existing, so I changed it to `validate_app_full`.  Is this the correct change to make?